### PR TITLE
Music notification

### PR DESCRIPTION
**Pull request by Mrinalsinghal**

### CODE REVIEW

File: app/assets/javascripts/front/music-player.js [hunk 1]
```diff
@@ -126,4 +131,48 @@ class MusicPlayer {
...
+      navigator.mediaSession.metadata = new MediaMetadata({
+        title: data.name,
+        artist: this.artistsList.innerText,
+        artwork: [
+          { src: 'https://dummyimage.com/96x96', sizes: '96x96', type: 'image/png' },
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
```
Comment (Ardnived) [inline app/assets/javascripts/front/music-player.js:154]: I see that you are using placeholder images right now. What did you implementation using `data.cover_image_url` look like?

File: app/assets/javascripts/front/music-player.js [hunk 2]
```diff
@@ -126,4 +131,48 @@ class MusicPlayer {
...
+        Amplitude.prev()
+      })
+
+      navigator.mediaSession.setActionHandler('nexttrack', () => {
+        Amplitude.next()
+        Amplitude.play()
+      })
+
+      navigator.mediaSession.setActionHandler('seekbackward', () => {
+        this.seek(skipSeconds, 'backward')
```
Comment (Ardnived) [inline app/assets/javascripts/front/music-player.js:168]: Where does the `skipSeconds` variable come from?

File: app/assets/javascripts/front/music-player.js (hunk 1 above)
Comment (Mrinalsinghal) [reply app/assets/javascripts/front/music-player.js:154]: Simply like `src: data.cover_art_url`.
The `data` is like this:
```
artists: Array(2)
    0: {name: "Leo Vertunni", url: "http://indialucia.com"}
    1: {name: "Shivakumar Sharma", url: "http://www.santoor.com/"}
cover_art_url: "/uploads/artist/image/5/39e74f12ec.jpg"
duration: null
index: 0
live: false
name: "Raag Durga"
url: "/uploads/track/1/music.mp3"
```

File: app/assets/javascripts/front/music-player.js (hunk 2 above)
Comment (Mrinalsinghal) [reply app/assets/javascripts/front/music-player.js:168]: line 143

**Comment by Ardnived** [comment]
I have added some code which should allow you to render all the different versions of the image that we have for the media session api - but I have not tested it (my phone is not set up to test with).

**Pull request merged**